src/components/swap/TradePrice.tsx

```diff
@@ -27,8 +27,8 @@ export default function TradePrice({
 
   const show = Boolean(inputCurrency && outputCurrency)
   const label = showInverted
-    ? `${outputCurrency?.symbol} per ${inputCurrency?.symbol}`
-    : `${inputCurrency?.symbol} per ${outputCurrency?.symbol}`
+    ? `${outputCurrency?.symbol} / ${inputCurrency?.symbol}`
```

Comment:
here as well? will the / make sense in every language?

Reply:
I think mathematics should be more general than words, how do you think about it?